### PR TITLE
Combined both tasks for AmazonLinux into one block.

### PR DESCRIPTION
**Pull request by mcdonnnj**
Per a conversation with @jsf9k this change will combine both AmazonLinux tasks into a single block statement to consolidate functionality.

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -22,12 +14,21 @@
...
-      - python36-pip
-      - python36-devel
+  block:
+    # This is required temporarily, because of
+    # https://github.com/ansible/ansible/issues/56583
+    - name: Temporary workaround for ansible/ansible#56583
+      set_fact:
+        ansible_facts:
+          pkg_mgr: yum
+
```
Comment (jsf9k) [inline tasks/main.yml]: Let's remove this blank line, just so the block is visually a contiguous unit.

**Review by jsf9k — APPROVED** [review]
I had one very minor suggestion that you can take or leave.  Thanks for doing this...I had already forgotten about it!

File: tasks/main.yml (hunk 1 above)
Comment (mcdonnnj) [reply tasks/main.yml]: Sure, I wasn't sure which would be preferred.

**Review by dav3r — APPROVED** [review]
Nice! 👍

**Pull request merged**